tests/xmap_test.py

```diff
@@ -302,6 +307,28 @@ def f(x, y):
 
     self.assertAllClose(z, jnp.einsum('nij,njk->nik', x, y))
 
+  @parameterized.named_parameters(
+    {"testcase_name": name, "mesh": mesh, "axis_resources": axis_resources}
+    for name, mesh, axis_resources in (
+      ('', (), ()),
+      ('Mesh', (('x', 2),), (('i', 'x'),))
+    ))
+  @ignore_xmap_warning()
+  def testMultipleCalls(self, mesh, axis_resources):
+    def f(x, y):
+      return x * y
+
+    @with_mesh(mesh)
+    def run_test():
+      f_mapped = xmap(f,
+                      in_axes=(['i', ...], ['j', ...]),
+                      out_axes=['i', 'j', ...],
+                      axis_resources=dict(axis_resources))
```

Comment:
If the test is using axis_resources / mesh then it's not actually going to exercise the problematic branch, is it?

Reply:
Adam - sorry I just misread the conditions the first time!